src/main/java/com/google/devtools/build/lib/actions/ResourceSet.java

```diff
@@ -103,16 +118,25 @@ public double getCpuUsage() {
     return cpuUsage;
   }
 
+  public Map<String, Float> getExtraResourceUsage() {
+    return extraResourceUsage;
+  }
+
   /** Returns the local test count used. */
   public int getLocalTestCount() {
     return localTestCount;
   }
 
   @Override
   public String toString() {
+    StringBuilder sb = new StringBuilder();
```

Comment:
You can use Joiner.MapJoiner for a more readable version.

Reply:
Done.